ticdc/ticdc-open-api.md

```diff
@@ -0,0 +1,617 @@
+---
+title: TiCDC Open API
+summary: Learn how to use the Open API interface to manage the cluster status and data replication.
+---
+
+# TiCDC Open API
+
+<!-- markdownlint-disable MD024 -->
+
+> **Warning:**
+>
+> TiCDC Open API is still an experimental feature. It is not recommended to use it in a production environment.
+
+TiCDC provides the Open API feature for querying and operating the TiCDC cluster, which is similar to that of [`cdc cli` tool] (/ticdc/manage-ticdc.md#use-cdc-cli-to-manage-cluster-status-and-data-replication-task).
+
+You can use the Open APIs to perform the following maintenance operations on the TiCDC cluster:
+
+- [Get the status information of a TiCDC node](#get-the-status-information-of-a-ticdc-node)
+- [Check the health status of a TiCDC cluster](#check-the-health-status-of-a-ticdc-cluster)
+- [Create a replication task](#create-a-replication-task)
+- [Remove a replication task](#remove-a-replication-task)
+- [Update the replication configuration](#update-the-replication-configuration)
+- [Query the replication task list](#query-the-replication-task-list)
+- [Query a specific replication task](#query-a-specific-replication-task)
+- [Pause a replication task](#pause-a-replication-task)
+- [Resume a replication task](#resume-a-replication-task)
+- [Query the replication subtask list](#query-the-replication-subtask-list)
+- [Query a specific replication subtask](#query-a-specific-replication-subtask)
+- [Query the TiCDC service process list](#query-the-ticdc-service-process-list)
+- [Evict an owner node](#evict-an-owner-node)
+- [Manually trigger the load balancing of a table](#manually-trigger-the-load-balancing-of-a-table)
```

Comment:
It is not a table. It is all tables in a replication task.

Reply:
```suggestion
- [Manually trigger the load balancing of all tables in a replication task](#manually-trigger-the-load-balancing-of-all-tables-in-a-replication-task)
```